packages/dx-react-grid-material-ui/src/templates/table-header-cell/grouping-control.test.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { createShallow, getClasses } from 'material-ui/test-utils';
+import { GroupingControl } from './grouping-control';
+
+const defaultProps = {
+  onGroup: () => {},
+  align: 'left',
+};
+
+describe('GroupingControl', () => {
+  let shallow;
+  let classes;
+  beforeAll(() => {
+    shallow = createShallow({ dive: true });
+    classes = getClasses(<GroupingControl {...defaultProps} />);
+  });
+  afterAll(() => {
+    shallow.cleanUp();
```

Comment:
It appears that there is no need to clean up `shallow`. Only `mount` requires it.